Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/MotionWindow/MotionExtractionWindow.cpp

```diff
@@ -463,10 +465,23 @@ namespace EMStudio
     {
         const CommandSystem::SelectionList& selectionList = GetCommandManager()->GetCurrentSelection();
         const size_t numSelectedMotions = selectionList.GetNumSelectedMotions();
+        EMotionFX::RootMotionExtractionData* firstMotionRootExtractionData = nullptr;
 
-        if (numSelectedMotions == 1)
+        for (size_t motionIndex = 0; motionIndex < numSelectedMotions; ++motionIndex)
         {
-            EMotionFX::Motion* curMotion = selectionList.GetMotion(0);
+            EMotionFX::Motion* curMotion = selectionList.GetMotion(motionIndex);
+
+            // When multi selected, take the first motion's root motion data and copy it to other selected motions.
+            if (motionIndex == 0)
+            {
+                firstMotionRootExtractionData = curMotion->GetRootMotionExtractionData().get();
+            }
+            else
+            {
+                auto curMotionRootExtractionData =
+                    firstMotionRootExtractionData? AZStd::make_shared<EMotionFX::RootMotionExtractionData>(*firstMotionRootExtractionData) : nullptr;
```

Comment:
you can fix the formatting of this line
```suggestion
                    firstMotionRootExtractionData ? AZStd::make_shared<EMotionFX::RootMotionExtractionData>(*firstMotionRootExtractionData) : nullptr;
```